.github/workflows/build.yml

```diff
@@ -55,3 +55,27 @@ jobs:
           if-no-files-found: error
           path: |
             ./*/build/distributions/*.zip
+
+  generate-dokka:
```

Comment:
Lets split this up and only generate during a `release`, see https://github.com/getsentry/sentry-java/blob/main/.github/workflows/generate-javadocs.yml